src/authy/core.clj

```diff
@@ -15,12 +14,11 @@
   token-info)
 
 (defn- format-token-info
-  [token-info]
-  (assoc (dissoc token-info :token_type :expires_in :refresh_token :access_token)
-    :token-type    (:token_type token-info)
-    :expires-at    (Timestamp. (+ (System/currentTimeMillis) (* 1000 (:expires_in token-info))))
-    :refresh-token (:refresh_token token-info)
-    :access-token  (:access_token token-info)))
+  [{:keys [access_token refresh_token]}]
+  {:token-type    "bearer"
```

Comment:
I don't see the `token_type` field in the Tapis v3 response now, so I'm currently hard-coding "bearer" here, since the apps service is checking for this field:
https://github.com/cyverse-de/apps/blob/669e13b94c32027ad8eaaaa546f371ab3c0965d5/src/apps/persistence/oauth.clj#L10-L15

I'm not sure if we want to keep that check in the apps service, or if we should remove this type field.

Reply:
You're right that this library is only used for Tapis. Since the field is no longer being returned by their authentication system, we can probably simply remove the field from the result.

Reply:
Thanks, that sounds good to me.
So I'll update this PR and apps to remove the unnecessary field.